elide-core/src/main/java/com/yahoo/elide/core/RequestScope.java

```diff
@@ -65,6 +65,7 @@
     @Getter private final AuditLogger auditLogger;
     @Getter private final Optional<MultivaluedMap<String, String>> queryParams;
     @Getter private final Map<String, Set<String>> sparseFields;
+    @Getter private final Map<String, Map<String, Object>> requestedSubgraph;
```

Comment:
I think it would be good if we had a bit more structure to this.

Maybe we can make a class called FieldSet:

```java
class FieldSet {
  private Set<String> attributes;
  private Map<String, FieldSet> relationships;
}```